cli_tools

```diff
@@ -10,6 +10,25 @@
 : ${CSCONFIG_FILE:="/azure/ServiceConfiguration.Cloud.cscfg"}
 : ${AZURE_CLI_LOCATION:="/azure"}
 
+_retry() {
+  sleepsec=$1;
+  maxret=$2;
+  shift 2;
+  ret=1;
+  while :; do
+    [[ $ret > $maxret ]] && return 1;
+    echo "${ret}.retry...";
```

Comment:
please write to stderr. i know this strictscript doesn't really uses stderr. but this way the function is pretty much